core/modules/layout/layout.admin.inc

```diff
@@ -7,6 +7,70 @@
  */
 
 /**
+ * Output a page to enable or disable layout templates.
+ */
+function layout_settings_page($form, &$form_state) {
+  // Get the list of layout options.
+  $config = config('layout.settings');
+  $available_layouts = $config->get('templates');
+
+  $form['#attached']['js'][] = backdrop_get_path('module', 'layout') . '/js/layout.admin.js';
+  $form['#attached']['css'][] = backdrop_get_path('module', 'layout') . '/css/layout.admin.css';
+
+  $form['markup'] = array(
+    '#markup' => t('Only templates selected here will be enabled on layout settings pages.'),
+  );
+  $form['templates'] = array(
+    '#title' => t('Templates'),
+    '#type' => 'checkboxes',
+    '#default_value' => $available_layouts,
+    '#options' => array(),
+    '#wrapper_attributes' => array('class' => array('clearfix', 'layout-options')),
+  );
+  $form['show_all_admin'] = array(
+    '#title' => t('Show all for admins'),
+    '#description' => t('When enabled, all templates will be shown on layout edit pages for users with \'use all templates\' permissions.'),
```

Comment:
We seem to be using double quotes elsewhere in help text. For example in https://github.com/backdrop/backdrop/blob/1.x/core/modules/layout/layout.admin.inc#L243:

`Use the "%" (percent) symbol to indicate a wildcard in the path. i.e. "node/%" or "node/%/edit".`